internal/compiler/generator/cpp.rs

```diff
@@ -2776,6 +2776,21 @@ fn compile_expression(expr: &llr::Expression, ctx: &EvaluationContext) -> String
                 )
 
         }
+        Expression::MinMax { ty, op, lhs, rhs } => {
+            let ident = match op {
+                MinMaxOp::Min => "min",
+                MinMaxOp::Max => "max",
+            };
+            let lhs_code = return_compile_expression(lhs, ctx, Some(&ty));
+            let rhs_code = return_compile_expression(rhs, ctx, Some(&ty));
+            format!(
+                r#"[&]() -> {} {{ using namespace std; return {}({}, {}); }}()"#,
+                ty.cpp_type().unwrap_or_else(|| "void".to_string()),
+                ident,
+                lhs_code,
+                rhs_code
+            )
```

Comment:
I would just do this:

```suggestion
            format!("std::{indent}({lhs_code}, {rhs_code})")
```

No need to use `using namespace`  since we don't need (or want) ADL here.
So also no need for a namespace.

Now, it is possible that this is ambiguous. So perhaps it needs to be

```rust
format!("std::{indent}<{ty}>({lhs_code}, {rhs_code})", ty = ty.cpp_type().unwrap_or_default())
```

Reply:
Done